app/models/project.rb

```diff
@@ -6,6 +6,7 @@ class Project < ApplicationRecord
   belongs_to :demo_night
 
   validates_presence_of :name
+  validates_presence_of :description
```

Comment:
would validation be necessary? what if the project was self explanatory?

Reply:
I think the issue is that some student is either a) lazy and just doesn't want to fill it out or, b) thinks it's self-explanatory and either way Emma then has to go ask them for it which is what we're trying to avoid in the first place.